test/_interface/test_pymol_interface.py

```diff
@@ -67,3 +67,16 @@ def test_select_pymol_obj():
         1965, 1966, 1967, 1968, 1969, 1970, 1971, 1972, 1973, 1974, 1975, 1976, 1977,
         1978, 1979, 1980, 2009, 2229, 2618, 2619, 2620, 2622, 2623, 2624, 2626, 2732, 2733
     ]
+
+
+def test_point_mutate():
+    """testing that point mutations work as expected"""
+    test_stru = PDBParser().get_structure(f"{DATA_DIR}KE_07_R7_2_S.pdb")
+    test_mut_stru = PDBParser().get_structure(f"{DATA_DIR}KE_07_R7_2_S_mut.pdb")
+    pi = interface.pymol
+    test_session = pi.new_pymol_session()
+    pymol_obj_name, session = pi.load_enzy_htp_stru(test_stru, test_session)
+    pi.point_mutate(("A", 154), "TRP", pymol_obj_name, session)
+    assert len(pi.select_pymol_obj("resi 154", pymol_obj_name, session)) == (
```

Comment:
I found the reason why the atom index are different. Pymol add all the new atoms at the end. You can try the `sort` command in pymol about this. (It sorts when saving by default tho)

Reply:
<img width="372" alt="image" src="https://github.com/ChemBioHTP/EnzyHTP/assets/36098720/44ecb80d-7aee-40ae-8449-e28ad1b36554">

Reply:
![image](https://github.com/ChemBioHTP/EnzyHTP/assets/68860396/50404bdf-a613-4eab-a032-896db19252e5)

Even with sort, it looks like the atom indices are still off. Since it sorts when saving by default, though, should I keep the test the same for now?